internal/test/test_tpm.go

```diff
@@ -1,13 +1,15 @@
 package test
 
 import (
+	"encoding/binary"
 	"io"
 	"sync"
 	"testing"
 
 	"github.com/google/go-attestation/attest"
 	"github.com/google/go-tpm-tools/simulator"
 	"github.com/google/go-tpm/legacy/tpm2"
+	gotpm "github.com/google/go-tpm/tpm2"
```

Comment:
Why did you rename the import here? Generally avoid this: https://google.github.io/styleguide/go/decisions#import-renaming

Reply:
Since tpm2 is imported from 2 different packages, so I have added alias to one of tpm2 import. I have renamed the alias to gtpm2 to avoid any confusion.